sympy/printing/pretty/pretty.py

```diff
@@ -1983,6 +1983,12 @@ def _print_Tr(self, p):
         pform = prettyForm(*pform.right(')'))
         return pform
 
+    def _print_VectorExpressions(self, expr):
+        # XXX works only for applied functions
+        func_name = func.__name__
+        pform = self._print_Symbol(Symbol(func_name))
```

Comment:
@moorepants I have written the unit tests, but I'm facing some issues in the pretty printing part, can you help in pointing out the problem.